entries/l/lnk.bio.json

```
@@ -0,0 +1,12 @@
+{
+  "Lnk.Bio": {
+    "domain": "lnk.bio",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.lnk.bio/en/articles/8372705-how-to-add-2fa-security",
```

Only this part of the URL is needed :wink: 
Have any recovery instructions as well that could be linked to using the `recovery` element?
```suggestion
    "documentation": "https://help.lnk.bio/en/articles/8372705",
```